src/widget/wtracktableview.cpp

```diff
@@ -448,6 +448,10 @@ void WTrackTableView::createActions() {
     connect(m_pFileBrowserAct, SIGNAL(triggered()),
             this, SLOT(slotOpenInFileBrowser()));
 
+    m_pFileRemoveFromDiskAct = new QAction(tr("Remove from Disk"),this);
```

Comment:
The Remove word is already used. The file does not necessary stored on a disk.
How about "Delete from file system"

Reply:
Good point.

why not just 'Delete File'?

Reply:
I'd prefer "Delete file". I'm more concerned about users not understanding what a "file system" or "disk" is than a "file".

Reply:
I like "Delete File" or "Move To Trash"